README.md

```diff
@@ -98,7 +100,7 @@ let options = {platform: "pc"};
 const WF = new Warframe(options);
 
 (async function(){
-    let alerts = await WF.alerts;
+    let alerts = await WF.alerts();
```

Comment:
I don't think this is accurate?

Reply:
The getters aren't function invocations

Reply:
Huh, you're right, don't know why I changed that.